### PR TITLE
[SMALLFIX] Try to use LocalTachyonClusterResource() constructor

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: tests/src/test/java/tachyon/underfs/UnderStorageSystemInterfaceIntegrationTest.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
```
Comment (aaudiber) [inline tests/src/test/java/tachyon/underfs/UnderStorageSystemInterfaceIntegrationTest.java]: public final class?

**Comment by aaudiber** [comment]
There's a TODO on the no-arg constructor which can be removed with this PR

**Comment by aaudiber** [comment]
Thanks for improving consistency! I just left a couple minor comments

**Comment by jsimsa** [comment]
LGTM

File: tests/src/test/java/tachyon/underfs/UnderStorageSystemInterfaceIntegrationTest.java (hunk 1 above)
Comment (apc999) [inline tests/src/test/java/tachyon/underfs/UnderStorageSystemInterfaceIntegrationTest.java]: done

**Comment by apc999** [comment]
@aaudiber  there is still some followup work to fully address your TODO note. I will remove your TODO after that. Thanks for reminding me.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**